src/ubuntu/16.04/crossdeps/Dockerfile

```diff
@@ -34,13 +34,3 @@ RUN apt-get update \
         llvm-9 \
         python-lldb-6.0 \
     && rm -rf /var/lib/apt/lists/*
-
-# Install x86_arm crossgen dependencies
-RUN dpkg --add-architecture i386 \
```

Comment:
Are you sure we don't need it anymore? I have though that we use x86 crossgen to crossgen stuff for ARM.

Reply:
Pretty sure, we should generate arm from an x64 build @echesakovMSFT to confirm.

Reply:
Yes, on Linux we are running x64-arm crossgen, so we don't need this dependency.